--- a/plugins/builtin/include/content/providers/file_provider.hpp
+++ b/plugins/builtin/include/content/providers/file_provider.hpp
@@ -44,6 +44,7 @@ namespace hex::plugin::builtin {
         void setPath(const std::fs::path &path);

         [[nodiscard]] bool open() override;
+        bool open(bool memoryMapped);
         void close() override;

         void loadSettings(const nlohmann::json &settings) override;
@@ -65,8 +66,6 @@ namespace hex::plugin::builtin {
     private:
         void handleFileChange();

-        bool open(bool memoryMapped);
-
     protected:
         std::fs::path m_path;
         wolv::io::File m_file;
